docs/source/guides/getting_started/first_config.rst

```diff
@@ -0,0 +1,33 @@
+Initial Configuration
+=====================
+
+Your first configuration will look pretty simple:
+
+.. code-block:: yaml
+    :linenos:
+
+    configuration:
+        working_directory: '.ytdl-sub-downloads'
+
+    presets:
+        "My Favorite YouTube Channels":
+            preset:
+                - "Jellyfin TV Show by Date"
+                - "Only Recent"
+
+            overrides:
+                tv_show_directory: "/tv_shows/youtube"
+
+
+The first two lines in this ``config.yaml`` file are the ``configuration``, and define the ``working_directory``, which is described near the bottom of :ref:`this section <guides/getting_started/index:quick overview of \`\`ytdl-sub\`\`>`
+
+
+Line 4 begins the definition o your custom ``presets``, with line 5 being the name of your first custom ``preset``.
+
+Lines 7 and 8 tell ``ytdl-sub`` which :doc:`/guides/getting_started/prebuilt_presets` to expand on; these ``presets`` already indicate that the downloaded files should be: 
```

Comment:
In the docs, its noted as `Example Configs` when it should be `Prebuilt Presets`